stdlib/csv/from_test.go

```diff
@@ -7,16 +7,181 @@ import (
 	"time"
 
 	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
 	"github.com/influxdata/flux/execute"
 	"github.com/influxdata/flux/execute/executetest"
 	_ "github.com/influxdata/flux/fluxinit/static" // We need to init flux for the tests to work.
+	"github.com/influxdata/flux/internal/errors"
 	"github.com/influxdata/flux/mock"
 	"github.com/influxdata/flux/querytest"
 	"github.com/influxdata/flux/stdlib/csv"
 	"github.com/influxdata/flux/stdlib/universe"
 	"github.com/influxdata/flux/values"
 )
 
+func TestFromCSV_ReturnSingleResult(t *testing.T) {
+	tests := []struct {
+		name    string
+		spec    *csv.FromCSVProcedureSpec
+		want    []*executetest.Table
+		wantErr error
+	}{
+		{
+			name: "Happy path",
```

Comment:
Standard is either to use `HappyPath` (CamelCase) or `happy path` (lowercase sentence). I usually prefer the latter.